hw2/04-background.html

```diff
@@ -6,23 +6,38 @@
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
+  <div id="background">
   <body>
+  </div>
     <main>
-      <h1>Exercise 04 - Background</h1>
-      <p>
-        The background color will change every few seconds (as indicated below).
-        You can stop this behavior and you can also change how often the
-        background color changes.
-      </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
-      </div>
-      <div>
-        <input />
+      <div class="container my-5">
+        <div
+          class="card bg-light mb-5 mx-auto"
+          style="max-width: 30rem; height: 18rem"
+        >
+          <h3 class="my-3 px-3"><strong>Exercise 04 - Background</strong></h1>
```

Comment:
I think by mistake you opened a `<h3> `tag but while closing you mentioned it as `<h1>`